test/xpu/extended/run_test_with_skip.py

```diff
@@ -151,15 +151,16 @@
 )
 
 
-skip_options = " -k 'not " + skip_list[0]
+skip_options = " -k \"not " + skip_list[0]
 for skip_case in skip_list[1:]:
     skip_option = " and not " + skip_case
     skip_options += skip_option
-skip_options += "'"
+skip_options += "\""
 
-test_command = "PYTORCH_TEST_WITH_SLOW=1 pytest -v test_ops_xpu.py"
-test_command += skip_options
+os.environ["PYTORCH_ENABLE_XPU_FALLBACK"]="1"
```

Comment:
The extended test does not expect a fallback to cpu, it is to cover all the dtypes of the op we supported. Please remove this line.

Reply:
Thanks for the catch, done.